lib/Runtime/Library/IntlEngineInterfaceExtensionObject.cpp

```diff
@@ -446,7 +445,12 @@ namespace Js
             // Clear disable implicit call bit as initialization code doesn't have any side effect
             Js::ImplicitCallFlags saveImplicitCallFlags = scriptContext->GetThreadContext()->GetImplicitCallFlags();
             scriptContext->GetThreadContext()->ClearDisableImplicitFlags();
-            JavascriptFunction::CallRootFunctionInScript(function, Js::Arguments(callInfo, args));
+
+            {
+                AutoSuspendDebuggerRecording autoSuspendDebuggerRecording(scriptContext);
```

Comment:
Moved the class to ScriptContext.h which has all the types needed. Renamed it to AutoInitLibraryCodeScope.

---
In reply to: [129987339](https://github.com/Microsoft/ChakraCore/pull/3447#discussion_r129987339) [](ancestors = 129987339)